_data/events.yml

```diff
@@ -1287,3 +1287,13 @@
     core, proxies, plugins, and more.  As always, you're welcome to participate
     in the show, either via YouTube Live chat, or via our IRC channel
     (Freenode/#theforeman).
+- title: 'Foreman Usability - Red Hat Subscriptions page (with Walden Raines)'
+  type: 'Livestream'
+  location: virtual
+  start: 2018-04-04 14:00 BST
+  end: 2018-04-04 15:00 BST
+  url: 'https://www.youtube.com/watch?v=635AqkBIsTU'
+  description: |
+    The Red Hat Subscriptions page is being moved to react and changes will be made to allow a user to manage the number of subscription entitlements from within Katello rather than going to the Red Hat Portal.
```

Comment:
Could you break this into shorter lines?

Reply:
I'll do this on merge, no worries.